### PR TITLE
Add quiz component

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

Adds a basic quick component with functionless button.

### Demo

Screen Shot 2019-04-30 at 4 49 46 PM

Button styling lifted from the designs in Abstract

## Testing Instructions

Netlify is sufficient.

### CODE REVIEW

File: src/app/src/css/quiz.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.quiz__description {
+    width: 500px;
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+.quiz__button {
+    border-radius: 15px 15px 15px 15px;
```
Comment (maurizi) [inline src/app/src/css/quiz.css]: This can be simplified to:
```suggestion
    border-radius: 15px;
```

**Review by maurizi — APPROVED** [review]
:+1: I left a comment on simplifying the CSS, but I don't think I need to do another review after you tackle that.

File: src/app/src/css/quiz.css (hunk 1 above)
Comment (fungjj92) [reply src/app/src/css/quiz.css]: 🤦‍♀

**Comment by fungjj92** [comment]
@maurizi @alexelash Tried styled-components here as well. 🤔 I like JS in CSS, but am not totally satisfied with this implementation. I think with more styling rules in bigger components, it will be clearer how to bucket rules into styled components.

File: src/app/src/Quiz.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+`;
+
+const Button = styled.button`
+    border-radius: 15px;
+    box-shadow: 0px 4px 12px 0px rgba(17, 20, 39, 0.9);
+    height: 70px;
+    width: 200px;
+`;
+
+const Text = styled.div``;
```
Comment (fungjj92) [inline src/app/src/Quiz.js]: I'll change to `p` element

**Pull request merged**